Library/Homebrew/release_notes.rb

```diff
@@ -25,8 +25,12 @@ def generate_release_notes(start_ref, end_ref, markdown: false)
 
     if markdown
       log_output.map! do |s|
-        /(.*\d)+ \(@(.+)\) - (.*)/ =~ s
-        "- [#{Regexp.last_match(3)}](#{Regexp.last_match(1)}) (@#{Regexp.last_match(2)})\n"
+        # (https://github.com/Homebrew/pull)/(1234) (@author) - (title)
+        # maps to
+        # [title #1234](https://github.com/Homebrew/pull/1234) (@author)
+        %r{(.*)/(\d+) \(@(.+)\) - (.*)} =~ s
+        "- [#{Regexp.last_match(4)} \##{Regexp.last_match(2)}](#{Regexp.last_match(1)}/#{Regexp.last_match(2)})" \
+        " (@#{Regexp.last_match(3)})\n"
```

Comment:
```suggestion
        matches =~ s.match %r{(?<url>.*)/(?<pr>\d+) \(@(?<user>.+)\) - (?<title>.*)}
        "- [#{matches[:title]} \##{matches[:pr]}](#{matches[:url]}/#{matches[:pr]})" (@#{matches[:user]})\n"
```

I think using named captures will help readability here

Reply:
your suggestions sound good to me; I'm about to go to bed, so please feel free to push to this branch as you see fit

Reply:
> I'm about to go to bed

Same here (should have done that hours ago) 🛌😴💤

I'll take a look when I'm available tomorrow if you haven't had a chance yet.